web/src/engine/FetchProviderNodeWebkit.ts

```diff
@@ -149,6 +154,24 @@ export async function FetchCSS<T extends HTMLElement>(request: FetchRequest, que
     return [...dom.querySelectorAll(query)] as T[];
 }
 
+export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, variables: string): Promise<TResult> {
+
+    const graphQLRequest = new FetchRequest(request.url, {
+        body: JSON.stringify({ operationName, query, variables }),
+        method: 'POST',
+        headers: { 'content-type': 'application/json', 'accept': '*/*' }
+    });
+
+    const data = await FetchJSON<GraphQLResult<TResult>>(graphQLRequest);
+    if (data.errors) {
```

Comment:
I copied the existing code from Hakuneko fetchgraphQl and it works fine with test plugins i made (HQnow and ArgoScans).

It seems errors is undefined when there is no errors 🤷 

Maybe its worth testing its length too just in case.